msal/src/main/java/com/microsoft/identity/client/Prompt.java

```diff
@@ -66,7 +71,7 @@ public OpenIdConnectPromptParameter toOpenIdConnectPromptParameter() {
         switch (this) {
             case SELECT_ACCOUNT:
                 return OpenIdConnectPromptParameter.SELECT_ACCOUNT;
-            case FORCE_LOGIN:
+            case LOGIN:
                 return OpenIdConnectPromptParameter.LOGIN;
             case CONSENT:
                 return OpenIdConnectPromptParameter.CONSENT;
```

Comment:
Shouldn't default value be `OpenIdConnectPromptParameter.NONE`

Reply:
It should be WhenRequired, but i'm not changing it in this PR.